### ISSUE TITLE
Rule 942500 is supposed to catch "/*+*/", but it does not

### ISSUE THREAD

**Issue by dune73** [coreruleset/coreruleset#3359]
### Description

False negative on 942500 with one of the examples given in comments (`# The minimal string that triggers this regexp is: /*!*/ or /*+*/.`)

### How to reproduce the misbehavior (-> curl call)

Test:

```
$ curl localhost -d 'foo=/*+*/'
```

Expected result: alert on 942500

Actual result: no alert

### Your Environment

* CRS version (e.g., v3.3.4): v4.0/dev

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by theseion** [comment]
The issue is the `urlDecodeUni` transformation, which converts the `+` into a space. I don't think that particular string was ever caught by rule 942500.

**Comment by Xhoenix** [comment]
The transformation [replaceComments](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#user-content-replaceComments) can be useful, and I checked it is used a few other SQLi rules. Is there any reason why we're not using it?

**Comment by theseion** [comment]
Not that I'm aware of. Might be a good idea.
